x-pack/test/alerting_api_integration/security_and_spaces/tests/alerting/update.ts

```diff
@@ -233,7 +233,7 @@ export default function createUpdateTests({ getService }: FtrProviderContext) {
                 statusCode: 400,
                 error: 'Bad Request',
                 message:
-                  'child "interval" fails because ["interval" with value "10x" fails to match the seconds pattern, "interval" with value "10x" fails to match the minutes pattern, "interval" with value "10x" fails to match the hours pattern, "interval" with value "10x" fails to match the days pattern]. "alertTypeId" is not allowed',
+                  'child "interval" fails because ["interval" with value "10x" fails to match the seconds (5s) pattern, "interval" with value "10x" fails to match the minutes (5m) pattern, "interval" with value "10x" fails to match the hours (5h) pattern, "interval" with value "10x" fails to match the days (5d) pattern]. "alertTypeId" is not allowed',
```

Comment:
nit: seems slightly weird to include the examples here (eg, `(5s)`), seems like a customer could be confused wondering where that came from

Reply:
Fixed in https://github.com/elastic/kibana/pull/41919/commits/f688eedb567b8889b553d95fc47cf29535879d7d.